Model/GetItemsToDeductFromOrder.php

```diff
@@ -52,7 +52,6 @@ public function execute(Order $order): array
     {
         $itemsToOrder = [];
 
-        /** @var \Magento\Sales\Model\Order\Item $orderItem */
```

Comment:
Umm, so what's the PHPStan issue here? We shouldn't be removing type hints if we can help it